### PR TITLE
Indico work so far -- not integrated

### PR DESCRIPTION
**Pull request by mary-keenan**

### CODE REVIEW

File: MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        final View view = inflater.inflate(R.layout.fragment_indico_experimenting, container, false);
+
+        final String APIKey = BuildConfig.API_KEY;
+        final TextView output = (TextView) view.findViewById(R.id.sampleOutput);
+        String url = "https://apiv2.indico.io/fer"; //url for Indico's facial emotion recognition
```
Comment (sammyers) [inline MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java]: This should probably be stored as an XML resource, not hardcoded.

File: MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+        StringRequest postRequest=new StringRequest(Request.Method.POST,url,new Response.Listener<String>() {
+            @Override
+            public void onResponse(String response) {
+                HashMap<String, Float> emotionDict = new HashMap(); //dictionary we store emotions and their values in
+                Float max_val = Float.parseFloat("0");
+                String max_emotion = "";
+                String shortenedResponse = response.substring(13, response.length()-2).replaceAll("\"",""); //strip "{"results": {", "}", and " from original response
```
Comment (sammyers) [inline MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java]: Not a fan of manually parsing JSON. Jackson and Gson are both good libraries for extracting this data.

File: MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                output.setText("You are " + max_emotion.toLowerCase()); //set emotion to text
+                Log.d("EDITED*****:",shortenedResponse);
+            }
+        }, new Response.ErrorListener() {
+            @Override
+            public void onErrorResponse(VolleyError volleyError) {
+                Log.d("error:",volleyError.toString());
+
+            }
+        }){@Override
```
Comment (sammyers) [inline MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java]: Formatting

**Review by sammyers — COMMENTED** [review]
There are some formatting tweaks that need to be made, and the JSON parsing should be redone.

File: MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java (hunk 1 above)
Comment (mary-keenan) [reply MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java]: Check

File: MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java (hunk 3 above)
Comment (mary-keenan) [reply MoodCookie/app/src/main/java/cecelia/moodcookie/IndicoCrap.java]: Check

**Comment by cbauerswald** [comment]
lgtm, cant wait to test

**Pull request merged**